package.json

```diff
@@ -50,6 +50,7 @@
     "jest-canvas-mock": "^1.1.0",
     "jshint-summary": "^0.4.0",
     "mini-css-extract-plugin": "^0.11.2",
+    "modules": "^0.4.0",
```

Comment:
Is this used?

Reply:
i don't think it is used, i can't find it being used.